federation/api/install/install.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
```

Comment:
If I understand what you're doing, I think you can do it much more simply in a single package rather than creating an entire package tree like this. Please don't call the package "api". :)